keylime/signing.py

```diff
@@ -0,0 +1,77 @@
+'''
+SPDX-License-Identifier: Apache-2.0
+Copyright 2017 Massachusetts Institute of Technology.
+'''
+
+import gnupg
+import os
+
+from cryptography.hazmat.primitives import hashes
+from cryptography.hazmat.primitives.asymmetric import ec
+from cryptography.hazmat.primitives.serialization import load_pem_public_key
+
+from keylime import keylime_logging
+
+logger = keylime_logging.init_logging('signing')
+
+def verify_signature_from_disk(key_file, filename, sig_file, file_description):
+    """
+       Read signing materials from disk and verify.
```

Comment:
May want to document the parameters like you did below: Verify the file signature (sig) using a public key (key) with the file (file).

Reply:
Done!